packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,30 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
+    slug: 'storybook',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/storybook.svg',
+    tagline: 'Frontend workshop for UI development',
```

Comment:
Are you missing the `demo` prop?

Reply:
`demo` is not required and I didn't prepare a demo. Can that be added in a future PR?